### ISSUE TITLE
Error with plot

### ISSUE THREAD

**Issue by RoelvReij** [choishingwan/PRSice#107]
Since a few days I get an error everytime I try to do a PRS and the error is related to the plotting:
 .External2(C_X11, paste0("png::", filename), g$width, g$height,  :
  unable to start device PNG
Calls: process_plot ... bar_plot -> plot.bar -> ggsave -> dev -> <Anonymous>
In addition: Warning message:
In grDevices::png(..., res = dpi, units = "in") :
  unable to open connection to X11 display ''
Execution halted
Could you tell me where it goes wrong? even when I try a PRS that I have already done in the past and from which I did get a good plot, the error appears.
my code is as follows:
Rscript PRSice.R --dir . \
    --prsice ./PRSice_linux \
    --base depression.txt \
    --target hys_GP_r2_geno_maf \
    --thread 1 \
    --binary-target T \
	--pheno-file hys_snptest_prs \
	--pheno-col hyspainhigh \
	--out prs_depression \
	--clump-kb 250 \
	--pearson 0.5 \
	--snp SNP \
	--seed 871469140 \
	--A1 A1 \
	--A2 A2 \
	--stat BETA \
	--se SE \
	--pvalue P \
	--model add \
	--print-snp

**Comment by RoelvReij** [comment]
This leads to the same error indeed. It seems that the X11 server is missing form the linux server I use.
I have now switched form putty to mobaXterm which apparantly has a build in X server and I does produce the test plot you mentioned. I will now try whether it also plots the PRS graphs

**Comment by RoelvReij** [comment]
unfortunately, my error was not completely solved yet.
When I tried it again, I got a different error:

Error in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
  X11 font -adobe-helvetica-%s-%s-*-*-%d-*-*-*-*-*-*-*, face 5 at size 18 could not be loaded
Calls: process_plot ... <Anonymous> -> widthDetails -> widthDetails.text -> grid.Call
Execution halted

I have checked for the fonts available in R and at the linux server and adobe-helvetica should be available

**Comment by choishingwan** [comment]
Will this help?

https://askubuntu.com/questions/449578/x11-font-adobe-helvetica-s-s-d-face-2-at-size-11-could-no


Dr Shing Wan Choi
Postdoctoral Fellow
Genetics and Genomic Sciences
Icahn School of Medicine, Mount Sinai, NYC

**Comment by rodrigoduarte88** [comment]
Hi all, I'm having this issue with PRSice2 too, but I realized it's a problem of R version, not of PRSice (see this post: https://stackoverflow.com/questions/56315182/how-should-i-solve-unable-to-open-connection-to-x11-display-this-problem-at-l ). When I type in "capabilities()" within the R version that was giving me the problem, I see png FALSE.

I've installed a new version of R in the cluster using conda ( conda create -n Renv r-base=3.6.1), and when I type "capabilities()" on this new R, it says PNG TRUE. 

The " unable to open connection to X11 display " error goes away.
